Unit3_Real/Unit3_Real/Controller/PeopleViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ViewController.swift
+//  Unit3_Real
+//
+//  Created by EricM on 9/2/19.
+//  Copyright © 2019 EricM. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
+class PeopleViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, UISearchBarDelegate {
+    @IBOutlet weak var userTableViewOutlet: UITableView!
+    @IBOutlet weak var searchOutlet: UISearchBar!
+    
+    var human = [Person]()
+    
```

Comment:
Good job with the search bar functionality. I think it would be more organized if you moved some of that logic into the model for filtering the results. Let me know if you need help with that.